third_party/amd/include/TritonAMDGPUTransforms/Passes.td

```diff
@@ -166,4 +166,21 @@ def TritonAMDGPUBlockPingpong: Pass<"tritonamdgpu-block-pingpong", "mlir::Module
   let dependentDialects = ["mlir::ROCDL::ROCDLDialect, mlir::triton::amdgpu::TritonAMDGPUDialect"];
 }
 
+def TritonAMDGPUInThreadTranspose: Pass<"tritonamdgpu-in-thread-transpose", "mlir::ModuleOp"> {
+  let summary = "Transpose K-outer dot operand while data is loaded into register right before writing to LDS";
```

Comment:
"Optimize global load to 2D manner and then perform transpose within registers per thread before writing to shared memory to make K the contious dimension"